### ISSUE TITLE
Rename aria label "Gutenberg tips" to not mention "Gutenberg"

### ISSUE THREAD

**Issue by ocean90** [WordPress/gutenberg#12436]
**Describe the bug**
There's only [one last reference](https://translate.wordpress.org/projects/wp/dev/en-au/default?filters%5Bterm%5D=gutenberg&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated) of "Gutenberg" in the UI which should be removed/replaced:

The label is used for these popovers:
<img width="484" alt="popover-tip" src="https://user-images.githubusercontent.com/617637/49236609-bf348b00-f3fc-11e8-8e54-5df166b1f9c8.png">

Source: https://github.com/WordPress/gutenberg/blob/76c629f/packages/nux/src/components/dot-tip/index.js#L41
